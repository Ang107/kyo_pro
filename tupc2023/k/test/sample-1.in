3
2 2 4
5 5 17
6 6 30
