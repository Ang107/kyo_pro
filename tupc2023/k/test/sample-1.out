Yes
1 2
3 4
No
Yes
22 9 31 11 33 32
23 10 2 25 3 19
20 21 8 1 30 13
36 29 16 17 24 7
12 35 27 14 18 34
26 6 28 15 5 4
